From 5c347e32099785039cdca83db224db8c123572ba Mon Sep 17 00:00:00 2001
From: md_5 <md_5@live.com.au>
Date: Sat, 23 Mar 2013 13:31:23 +1100
Subject: [PATCH] Entity Tracking Intervals

Send boats at the same speed as their player to remove the illusion of lag / jerkiness.
---
 src/main/java/net/minecraft/server/EntityTracker.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/java/net/minecraft/server/EntityTracker.java b/src/main/java/net/minecraft/server/EntityTracker.java
index 0bc3733..bed96f6 100644
--- a/src/main/java/net/minecraft/server/EntityTracker.java
+++ b/src/main/java/net/minecraft/server/EntityTracker.java
@@ -58,7 +58,7 @@ public class EntityTracker {
         } else if (entity instanceof EntityMinecartAbstract) {
             this.addEntity(entity, 80, 3, true);
         } else if (entity instanceof EntityBoat) {
-            this.addEntity(entity, 80, 3, true);
+            this.addEntity(entity, 80, 2, true); // Spigot - send at same speed as player
         } else if (entity instanceof EntitySquid) {
             this.addEntity(entity, 64, 3, true);
         } else if (entity instanceof EntityWither) {
-- 
1.8.1-rc2

